src/Components/Home/About/About.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "../../../scss/pages/_about.scss";
```

Comment:
Create a scss file in the `src/Components/Header` directory itself. Create scss files in scss directory only which are to be used globally.